src/main/java/com/minecolonies/apiimp/CommonMinecoloniesAPIImpl.java

```diff
@@ -382,27 +408,41 @@ public IForgeRegistry<QuestRegistries.DialogueAnswerEntry> getQuestDialogueAnswe
     }
 
     @Override
-    public IForgeRegistry<HappinessRegistry.HappinessFactorTypeEntry> getHappinessTypeRegistry()
+    public IForgeRegistry<EquipmentTypeEntry> getEquipmentTypeRegistry()
     {
-        return happinessFactorTypeRegistry;
+        return equipmentTypeRegistry;
     }
 
     @Override
-    public IForgeRegistry<HappinessRegistry.HappinessFunctionEntry> getHappinessFunctionRegistry()
+    @NotNull
+    public IMobAIRegistry getMobAIRegistry()
     {
-        return happinessFunctionRegistry;
+        return mobAIRegistry;
     }
 
     @Override
-    public IForgeRegistry<EquipmentTypeEntry> getEquipmentTypeRegistry()
+    @NotNull
+    public IPathNavigateRegistry getPathNavigateRegistry()
     {
-        return equipmentTypeRegistry;
+        return pathNavigateRegistry;
     }
 
     @Override
-    public EventBus getEventBus()
+    public IModelTypeRegistry getModelTypeRegistry()
     {
-        return eventBus;
+        return null;
+    }
+
+    @Override
+    public IGlobalResearchTree getGlobalResearchTree()
```

Comment:
Not really possible to review what changed here because of the re-ordering too

Reply:
Nothing changed, the PR does exactly what it means to do, it's a cleanup